hugegraph-pd/hg-pd-service/pom.xml

```diff
@@ -138,11 +144,13 @@
                 <executions>
                     <execution>
                         <configuration>
-                            <classifier>spring-boot</classifier>
                             <mainClass>
                                 org.apache.hugegraph.pd.boot.HugePDServer
                             </mainClass>
                         </configuration>
+                        <goals>
+                            <goal>repackage</goal>
+                        </goals>
```

Comment:
we need some comment here?

Reply:
> refer to [docs.spring.io/spring-boot/docs/current/maven-plugin/reference/htmlsingle/#packaging](https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/htmlsingle/#packaging)

fine, better to add url refer for it in future, currently leave this in PR context